fjord/feedback/tests/test_api.py

```diff
@@ -1,3 +1,5 @@
+# This Python file uses the following encoding: utf-8
+
```

Comment:
This isn't how you set the encoding. You need to do this:

```
# -*- coding: utf-8 -*-
```

You can see examples in the migration files.